test/components/user_list_item_test.js

```diff
@@ -45,6 +59,57 @@ describe("UserListItem", () => {
     })
   })
 
+  describe("passed a non-facilitator as a facilitator", () => {
+    const user = { ...defaultUserAttrs, is_facilitator: true }
```

Comment:
a nitpick, but you `user` -> `facilitator` and `user2` -> `nonFacilitator` would make this easier to read, as then readers don't need to remember that user one is the facilitator and user two isn't